_summaries/hedman2021baking_1.md

```diff
@@ -1,15 +1,18 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{hedman2021baking}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: timchen0618
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
+## What is the core idea?
+NeRF is a good method to recreate 3D scenes from viewpoints unobserved by the system. However, it is computationally intensive and cannot be applied in real-time. The authors proposed a novel approach to train NeRF called Sparse Neural Radiance Grid (SNeRG) and achieved real-time rendering. 
+
+## How is it realized (technically)?
+
 * How well does the paper perform?
```

Comment:
I think it is better if you can add more about the experiments results in section 7, which are about ablation studies, and baseline comparisons